### PR TITLE
[1.4.x] JBERET-548 switch to jberet-parent 1.4.7.Final

### PR DESCRIPTION
**Pull request by Zerrossetto**
Inherited definition of build-helper-plugin to fix Automatic-Module-Name
in MANIFEST.MF

### CODE REVIEW

**Comment by chengfang** [comment]
jberet-parent jberet-core jberet-se 1.4.7.Final has been released. Can you update the PR? @Zerrossetto

**Comment by Zerrossetto** [comment]
Done, please look at it now.

File: pom.xml [hunk 1]
```diff
@@ -18,11 +18,11 @@
     <parent>
         <groupId>org.jberet</groupId>
         <artifactId>jberet-parent</artifactId>
-        <version>1.4.5.Final</version>
+	<version>1.4.7.Final</version>
     </parent>
 
     <artifactId>jberet-support</artifactId>
-    <version>1.4.1.Final-SNAPSHOT</version>
+    <version>1.4.7.Final-SNAPSHOT</version>
```
Comment (chengfang) [inline pom.xml:25]: I think we should keep the sequence, and not skip directly to 1.4.7 for jberet-support version. So this line need no change.

Comment (Zerrossetto) [reply pom.xml]: Now it should be ok.

**Review by chengfang — APPROVED** [review]

**Pull request merged**